virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc

```diff
@@ -14,39 +14,26 @@ The Scheduling, Scale, and Performance (SSP) Operator integrates {VirtProductNam
 * Run commands in VMs
 * Manipulate disk images with `libguestfs` tools
 
-:FeatureName: Managing virtual machines with {pipelines-title}
-include::snippets/technology-preview.adoc[]
```

Comment:
This seems like it's changing a support statement by removing the TP snippet.

Reply:
Yes, this feature is no longer a TP.